Update version: RoyalApps.RoyalTS.V7 version 7.02.50424.0

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] This resolves #151770

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151787)

Dragon1573 committed:
Update version: RoyalApps.RoyalTS.V7 version 7.02.50424.0

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

stephengillie commented:
Hi @Dragon1573,

This manifest adds Apps and Features entries that aren't present in previous PR versions. These entries should be added to the previous versions, or removed from this version.

(Automated response - build 878.)
<!--
[Policy] Needs-Author-Feedback
-->

Trenly commented on manifests/r/RoyalApps/RoyalTS/V7/7.02.50424.0/RoyalApps.RoyalTS.V7.locale.en-US.yaml:
```diff
@@ -0,0 +1,23 @@
+Publisher: Royal Apps GmbH
+PublisherUrl: https://www.royalapps.com/
+PublisherSupportUrl: https://support.royalapps.com/support/home
+PackageName: Royal TS 7.02.50410.0
```
```suggestion
PackageName: Royal TS V7
```

Trenly commented on manifests/r/RoyalApps/RoyalTS/V7/7.02.50424.0/RoyalApps.RoyalTS.V7.installer.yaml:
```diff
@@ -0,0 +1,23 @@
+ProductCode: '{4959A733-614C-44E8-AC0D-3E7BCAA2F5AC}'
+ReleaseDate: 2024-04-24
+AppsAndFeaturesEntries:
+- DisplayName: Royal TS V7
+  DisplayVersion: 7.2.50424.0
+  UpgradeCode: '{4959A733-614C-44E8-AC0D-3E7B5A7375AB}'
```
```suggestion
- UpgradeCode: '{4959A733-614C-44E8-AC0D-3E7B5A7375AB}'
```

Dragon1573 committed:
Apply suggestions from code review

- Remove some parameters in AppsAndFeaturesEntries.
- Correction of PackageName.

All thanks to @Trenly

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

stephengillie commented:
Hi @Dragon1573,

This manifest adds Apps and Features entries that aren't present in previous PR versions. These entries should be added to the previous versions, or removed from this version.

(Automated response - build 880.)
<!--
[Policy] Needs-Author-Feedback
-->
